Billing.API/DopplerSecurity/PathSignatureOptions.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNetCore.Authentication;
+
+namespace Billing.API.DopplerSecurity
+{
+    public class PathSignatureOptions : AuthenticationSchemeOptions
+    {
+        public string QueryStringParameterName { get; set; } = "s";
```

Comment:
en mi PR lo puse con prefijo guión bajo, por si la `s` hay que usarla para otra cosa, o si se replica esto en algún otro lado

```suggestion
        public string QueryStringParameterName { get; set; } = "_s";
```